docs/content/preview/api/ysql/the-sql-language/statements/ddl_alter_table.md

```diff
@@ -182,6 +182,65 @@ It quietly succeeds. Now `\d children` shows that the foreign key constraint `ch
 
 Add the specified constraint to the table.
 
+#### [*alter_column_type*]
+
+Change the type of an existing column. The following semantics apply:
+- If data on disk is required to change, a full table rewrite is needed.
+- If the optional `COLLATE` clause is not specified, the default collation for the new column type will be used.
+- If the optional `USING` clause is not provided, the default conversion for the new column value will be the same as an assignment cast from the old type to the new type.
+- A `USING` clause must be included when there is no implicit assignment cast available from the old type to the new type.
+- Alter type is not supported on partitioned tables [#16980](https://github.com/yugabyte/yugabyte-db/issues/16980).
+- Alter type is not supported on tables with rules (limitation inherited from PostgreSQL).
+- Alter type is not supported on tables with CDC streams or xCluster replication when it requires data on disk to change [#16625](https://github.com/yugabyte/yugabyte-db/issues/16625).
+
+##### Alter type without table-rewrite
+
+If the change doesn't require data on disk to change, concurrent DMLs to the table can be safely performed.
+
+Example:
+
+```sql
+CREATE TABLE test (id BIGSERIAL PRIMARY KEY, a VARCHAR(50));
+ALTER TABLE test ALTER COLUMN a TYPE VARCHAR(51);
+```
+
+##### Alter type with table rewrite
+
+If the change requires data on disk to change, a full table rewrite will be done and the following semantics apply:
+- The action creates an entirely new table under the hood, and concurrent DMLs may not be reflected in the new table which can lead to correctness issues.
+- If the operation fails, it is possible that the existing table is renamed in DocDB. This may lead to issues like yb-admin commands that take table name.
```

Comment:
Unclear with this statement - "This may lead to issues like yb-admin commands that take table name."

Reply:
cc @fizaaluthra can you provide more info?

Reply:
I think it would read better if rephrased to: This may lead to issues with yb-admin commands that take table name. And optionally include an example of such a command (`./bin/yb-admin list_tablets`)

Reply:
Thanks @fizaaluthra . So @ddorian - "This may lead to issues with yb-admin commands that take table name. For example,`./bin/yb-admin list_tablets`"